java/managed-vm-gae/src/main/webapp/Dockerfile

```diff
@@ -13,14 +13,16 @@
 # limitations under the License.
 
 
-FROM b.gcr.io/jenkins_images/jetty9-compat
+FROM gcr.io/google-appengine/jetty9-compat
 
 # EDIT THESE VALUES -- Don't use quotes
 
 ENV BIGTABLE_PROJECT PROJECT_ID_HERE
 ENV BIGTABLE_CLUSTER CLUSTER_ID_HERE
 ENV BIGTABLE_ZONE    us-central1-b
-# ENV GOOGLE_APPLICATION_CREDENTIALS /app/WEB-INF/client-secret.json
+ENV GOOGLE_APPLICATION_CREDENTIALS /app/WEB-INF/client-secret.json
+
+ENV ALPN_ENABLE true
 
```

Comment:
All these env vars may be define in the appengine-web.xml in the 
  `<env-variables>`
  ` <env-var name="KKEY" value="VVALUE" />`
` </env-variables>`

section.
This way soon, you'll not need to define your own custom Dockerfile anymore.